qt-models/filtermodels.cpp

```diff
@@ -58,6 +58,7 @@ void FilterModelBase::updateList(const QStringList &newList)
 		anyChecked = true;
 	}
 	setStringList(newList);
+	emit dataChanged(createIndex(0, 0), createIndex(rowCount() - 1, 0));
```

Comment:
I'm not sure this is needed. The Qt documentation of `setStringList()` says:
> The model will notify any attached views that its underlying data has changed.

Which implies that it does all the necessary things..?

Reply:
Agree. I did not read the docs on setStringList() ...